admin/tabs/monitor/MemoryMonitorModel.js

```diff
@@ -0,0 +1,144 @@
+import {ChartModel} from '@xh/hoist/cmp/chart';
+import {dateTimeCol, GridModel} from '@xh/hoist/cmp/grid';
+import {HoistModel, LoadSupport, XH} from '@xh/hoist/core';
+import {FieldType} from '@xh/hoist/data';
+import {fmtTime, numberRenderer} from '@xh/hoist/format';
+import {checkMinVersion} from '@xh/hoist/utils/js';
+import {forOwn, sortBy} from 'lodash';
+
+@HoistModel
+@LoadSupport
+export class MemoryMonitorModel {
+
+    /** @member {GridModel} */
+    gridModel;
+    /** @member {ChartModel} */
+    chartModel;
+
+    get minVersionWarning() {
+        const minVersion = '8.7.0',
+            currVersion = XH.environmentService.get('hoistCoreVersion');
+        return checkMinVersion(currVersion, minVersion) ? null : `This feature requires Hoist Core v${minVersion} or greater.`;
+    }
```

Comment:
I know this is a bit elaborate, but I thought it was worth expressing that this is an optional admin feature that could be cleanly disabled if an app has an older version of HC - instead of incrementing the overall required version.

Reply:
a lighterweight/declarative way to do this would be just to add to the tab declaration in AppModel.js the following one-liner:

omit: !checkMinVersion(XH.environmentService.get('hoistCoreVersion') 8.7.0).

Just want to avoid cruft in the code from something like this...